mautic/mautic
# [UI] Marketplace extra info

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR is based [on another PR](https://github.com/mautic/mautic/pull/13735), where I introduce tokens to replace most of the UI variables, aiming to improve consistency in all components. **We need it to be merged before this one becomes available for test.**

It's also dependant of [this ](https://github.com/mautic/mautic/pull/13777) and [this](https://github.com/mautic/mautic/pull/13772) PRs.

#### Pending tasks

- [ ] Create Mautic shortlink for each link





![image](https://github.com/mautic/mautic/assets/116097999/9567ea55-642f-4867-8d4e-4b9a558cf6cc)


---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Marketplace > See
